projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -157,6 +158,13 @@ public function set_site_notices_menu_class( array $menu ) {
 		return $menu;
 	}
 
+	/**
+	 * Adds Inbox menu.
+	 */
+	public function add_inbox_menu() {
+		add_menu_page( __( 'Inbox', 'jetpack' ), __( 'Inbox', 'jetpack' ), 'edit_posts', 'https://wordpress.com/email/inbox/' . $this->domain, null, 'dashicons-email', '4.64424' );
```

Comment:
@AllTerrainDeveloper please point the menu to `/inbox` since we have resolved to switch to that high level route as per this [discussion](https://github.com/Automattic/wp-calypso/pull/56442#issuecomment-924692594).

Remember that we should only have one go at this.